### ISSUE TITLE
Google Logo (same question)

### ISSUE THREAD

**Issue by yayacannes** [julianlam/nodebb-plugin-sso-google#40]
Hello again,
so why I don't see your Google Logo here please ?

https://entreprendre.biz/login

problem of the material theme maybe ?

Thank you ;)

**Comment by julianlam** [comment]
Please do not open a new issue for follow-up questions, reply back to the original.

Try using the Persona theme and see, it could be the material theme does not respect the change.

**Comment by yayacannes** [comment]
I did because you closed the previous one...

**Comment by julianlam** [comment]
Yes, that's correct. It is closed as it is not a bug, but replies can be made because the issue is not locked.
